docs/ruleset_functions/get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM.md

```diff
@@ -0,0 +1,76 @@
+# get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM
+
+**Description:** Get the supply, return, exhaust, and relief total fan power and CFM associated with a fan system object.   The function returns a dictionary that saves the supply, return, exhaust, and relief fan power has a list and the saves the supply, return, exhaust, and relief cfm as a list {"Fan_Power": [supply fan power kW, return fan power kW, exhaust fan power kW, relief fan power kW], "Fan_CFM": [supply fan power CFM, return fan power CFM, exhaust fan power CFM, relief fan power CFM]}. Values will be equal to zero where not defined for a fan system. For CFM values this function assumes that all fans associated with the supply_fans object are in parallel (i.e., if multiple fans the cfm is additive.).
```

Comment:
Should we add a note to schema to aggregate fans of the same type so that there is always single supply, return, exhaust and relief fan per fan system?

Reply:
Shouldn't we also account for Terminal.fan flow and power?

Reply:
If more than 1 fan then outcome for pass or fail include note that it was assumed that the fans are in parallel and that the cf can be added.

Reply:
I added a check on the qty of fans and for pressure drop consistency. Would like to discuss on the call.